src/views/scans/showScansModal.tsx

```diff
@@ -92,21 +64,15 @@ const ShowScansModal: React.FC<ShowScansModalProps> = ({
       onClose={() => onClose()}
       {...(actions && { actions })}
     >
-      {(scanJobs && (
+      {(sortedScanJobs && (
         <React.Fragment>
           <div>
-            {scanJobs?.length} scan{scanJobs?.length === 1 ? ' has' : 's have'} run
+            {sortedScanJobs.length} scan{sortedScanJobs.length === 1 ? ' has' : 's have'} run
           </div>
           <br />
           <Table aria-label="Scan jobs table" ouiaId="scan_jobs_table">
             <Thead>
               <Tr>
-                {/* <Th
-                                    select={{
-                                        onSelect: (_event, isSelecting) => selectAllJobs(isSelecting),
-                                        isSelected: areAllJobsSelected
-                                    }}
-                                /> */}
                 <Th aria-labelledby="Sort by column" sort={getSortParams(0)}>
```

Comment:
Since we are introducing local sorting mechanism, and the table was not properly hooked to API requests for sorting, would you mind removing `sort` argument here?

```suggestion
                <Th aria-labelledby="Sort by column">
```

The same goes to line 79 (Scan Result header).

This will tie to the following issue from the doc:

> “Last Scanned” modal. Clicking table header items inside modal does not actually change the order of items in table.

Not fully a solution, but at least we won't present UI elements that wouldn't do anything.

Reply:
Mirek, I will introduce this change in a followup PR.

Reply:
Mirek, I will introduce this change in a followup PR.